### PR TITLE
Backend flags fix

### PR DESCRIPTION
**Pull request by gucio321**

### CODE REVIEW

File: backend.go [hunk 1]
```diff
@@ -15,7 +15,10 @@ import (
 	"unsafe"
 )
 
-var currentBackend Backend
+var currentBackend backendCExpose
+
+// TODO: Maybe we should get rid of it?
+var textureManager TextureManager
```
Comment (gucio321) [inline backend.go:21]: @AllenDang I'm having trouble here...
My solution is kinda stupid but can't find anything better.
Our texture stuff from `texture.go` needs access to backend in order to be loaded... And I don't want to make it debendent on BackendFlagsT :confused:
Maybe we should remove whole texture.go and let user implement it themselves?

**Comment by Wolf65** [comment]
Could you please add the GLFW_VISIBLE flag?

**Comment by gucio321** [comment]
@Wolf65 done

can i assume this will fix #194

**Comment by Wolf65** [comment]
Yes[ № 194](https://github.com/AllenDang/cimgui-go/issues/194) can be closed
maybe then so that it doesn't come back:

Floating Hint = C.GLFW_FLOATING // Specifies whether the window will be always-on-top.
Focused Hint = C.GLFW_FOCUSED // Specifies whether the window will be given input focus when created. This hint is ignored for full screen and initially hidden windows.
Iconified Hint = C.GLFW_ICONIFIED // Specifies whether the window will be minimized.

not sure about this one

AutoIconify Hint = C.GLFW_AUTO_ICONIFY // Specifies whether fullscreen windows automatically iconify (and restore the previous video mode) on focus loss.

File: backend.go (hunk 1 above)
Comment (AllenDang) [reply backend.go:21]: I prefer not to remove texture.go... like the default backend which could be removed by build via flag, maybe we could treat texture stuff as a bundle with backend, so to implement a backend, three things should be implemented, platform, render and texture.

Comment (gucio321) [reply backend.go:21]: ok, so lets leave it as-is for now. Gonna merge this

**Pull request merged**